shoop/xtheme/views/editor.py

```diff
@@ -83,10 +88,15 @@ def _populate_vars(self):
         theme = get_theme_by_identifier(self.request.GET["theme"])
         if not theme:
             raise Problem(_("Unable to determine current theme."))
+        view_name = self.request.GET["view"]
+        global_type = self.request.GET.get("global_type", None)
+        if global_type:
+            view_name = XTHEME_GLOBAL_VIEW_NAME
         self.view_config = ViewConfig(
             theme=theme,
-            view_name=self.request.GET["view"],
-            draft=True
+            view_name=view_name,
```

Comment:
This could be:
```
global_type = self.request.GET.get("global_type")
self.view_config = ViewConfig(
       theme=theme,
       view_name=(XTHEME_GLOBAL_VIEW_NAME if global_type else self.request.GET["view"]),
       draft=True,
       global_type=global_type
)
```